tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -184,59 +102,88 @@ $ mkdir -p ./proto/gogoproto
 $ curl https://raw.githubusercontent.com/cosmos/cosmos-sdk/v0.45.4/third_party/proto/gogoproto/gogo.proto -o ./proto/gogoproto/gogo.proto
 ```
 
-### Compilation
+Or you can add latest whole `third_party` folders by running `telescope install` under `myLib` folder:
 
-You can now compile the Protobuf files. To avoid adding all the `.proto` files manually to the command, use `xargs`:
+```sh
+telescope install
+```
+
+then select packages you want to include in `myLib/proto` folder:
 
-<CodeGroup>
+```txt
+Telescope 1.0.3
+? [pkg] which packages do you want to support? (Press <space> to select, <a> to toggle all, <i> to invert selection)
+❯◯ amino
+ ◯ akash
+ ◯ bcna
+ ...
+ ◯ cosmos
+ ...
+```
+
+Say `cosmos`'s selected and entered, there'll be cosmos folder and its dependencies(like gogoproto and google, etc) in `myLib/proto` folder.
```

Comment:
Is there a non-interactive command? If yes, then perhaps a `CodeGroup` is welcome with "interactive" and "non-interactive".